editor/views/record_sets.py

```diff
@@ -29,6 +29,29 @@
 ]
 
 
+def _generate_data(record_set: RecordSet) -> pd.DataFrame | None:
+    """Generates the first 3 records."""
```

Comment:
Is there any specific reason why 3? Would it make sense to parametrize it?

Reply:
Done.